RadixWallet/Features/HomeFeature/Coordinator/Home.swift

```diff
@@ -35,6 +35,7 @@ public struct Home: Sendable, FeatureReducer {
 
 	public enum InternalAction: Sendable, Equatable {
 		case accountsLoadedResult(TaskResult<Profile.Network.Accounts>)
+		case mnemonicAccessResult([FactorSourceID.FromHash: Bool])
```

Comment:
Bool is a bit ambiguous... at least typealias? Does true mean has access? Or true is is missing?